contracts/Killable.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity >=0.4.21 <0.6.0;
+
+import "./Ownable.sol";
+
+contract Killable is Ownable {
+  event LogKilled(address indexed account);
+
+  bool private killed;
+
+  modifier whenAlive() {
+    require(!killed, "Can't do that when the contract is killed!");
+    _;
+  }
+
+  function isKilled() public view returns (bool) {
+    return killed;
+  }
+
```

Comment:
Make this functionality part of the Pausable contract, we want to protect the user from accidentally calling this function by turning it into a 2 step-process:
1. The user pauses the contract
2. The user kills the contract (the kill function checks that the contract is paused)